docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -9,6 +9,7 @@ public static class Program
 
         internal static void RunAll()
         {
+            // Samples counter.
```

Comment:
What's the usefulness of this comment? If those words make a better name, it would be better to rename the variable with those words, than to add a comment like this.

Reply:
My bad, I meant to delete that. That was for an earlier test.